app/BUILD.bazel

```diff
@@ -57,6 +57,14 @@ LISTENER_FILES = [
     "src/main/java/org/oppia/app/utility/RegionClickEvent.kt",
 ]
 
+# Layout files for databinding_resources library.
+LAYOUT_RES_FILES = [
+    "src/main/res/layout-land/**",
```

Comment:
Does ``"src/main/res/layout*/**"`` work to replace all four?

Reply:
Yes, just replaced it!